plugins/inputs/mongodb/sample.conf

```
@@ -36,3 +36,9 @@
   # tls_key = "/etc/telegraf/key.pem"
   ## Use TLS but skip chain & host verification
   # insecure_skip_verify = false
+
+  ## Specifies plugin behavior regarding disconnected servers
+  ## Available choices :
+  ##   - default: telegraf will return an error on startup if one the servers is unreachable
+  ##   - retry: telegraf will start anyway and retry connexions to unreachable servers
+  # disconnected_servers_behavior = default
```

I would name it "error" to indicate what telegraf will do (just like "retry"). Also strings needs quotes.
```suggestion
  ##   - error: telegraf will return an error on startup if one the servers is unreachable
  ##   - retry: telegraf will start anyway and retry connexions to unreachable servers
  # disconnected_servers_behavior = "error"
```